src/ebay-confirm-dialog/confirm-dialog.tsx

```diff
@@ -40,24 +40,28 @@ const EbayConfirmDialog: FC<Props> = ({
             onCloseBtnClick={onReject}
         >
             {rest.children}
-            <EbayDialogFooter>
-                <EbayButton
-                    onClick={onReject}
-                    className="confirm-dialog__reject"
-                >
-                    {rejectText}
-                </EbayButton>
-                <EbayButton
-                    ref={confirmBtnRef}
-                    priority="primary"
-                    onClick={onConfirm}
-                    id={confirmId}
-                    aria-describedby={mainId}
-                    className="confirm-dialog__confirm"
-                >
-                    {confirmText}
-                </EbayButton>
-            </EbayDialogFooter>
+            {
+                (confirmText || rejectText) &&
+                <EbayDialogFooter>
+                    <EbayButton
```

Comment:
Should we render those button only when the text exists?

```jsx
(confirmText || rejectText) && 
<EbayDialogFooter>
  {rejectText && <EbayButton />}
  {confirmText && <EbayButton />}
</EbayDialogFooter>
```

In that way there is't a risk to render an empty button

Reply:
moved to button position logic